internal/dataplane/sendconfig/inmemory_schema.go

```diff
@@ -0,0 +1,124 @@
+package sendconfig
+
+import (
+	"github.com/kong/deck/file"
+)
+
+type ConsumerGroupConsumerRelationship struct {
+	ConsumerGroup string `json:"consumer_group"`
+	Consumer      string `json:"consumer"`
+}
+
+type ConsumerGroupPluginRelationship struct {
+	ConsumerGroup string `json:"consumer_group"`
+	Plugin        string `json:"plugin"`
+}
+
+type DBLessConfig struct {
+	file.Content                       `json:",inline"`
```

Comment:
```suggestion
	file.Content
```
I think it's redundant, btw. see https://github.com/golang/go/issues/6213